test/auth/actions/load-account-data-test.js

```diff
@@ -15,11 +15,13 @@ describe(`modules/auth/actions/load-account-data.js`, () => {
       const UpdateAssets = { updateAssets: () => {} }
       const UpdateLoginAccount = { updateLoginAccount: () => {} }
       const LoadAccountPositions = { loadAccountPositions: () => {} }
+      const approveAccounts = { checkAccountAllowance: () => {} }
```

Comment:
Suggest renaming to `ApproveAccounts` (to match the convention by the other mocks).